pkg/policy/mapstate.go

```diff
@@ -1193,6 +1210,8 @@ func (ms *mapState) AddVisibilityKeys(e PolicyOwner, redirectPort uint16, visMet
 				// 3. Change all L3/L4 ALLOW keys on matching port that do not
 				//    already redirect to redirect.
 				v.ProxyPort = redirectPort
+				v.priority = redirectPort
```

Comment:
This is a bit confusing -- why is the port the same as the priority?

Reply:
It's the other way around, by default the priority is the same as the redirect port number. This would normally be hidden in `NewMapStateEntry()` where this is also properly documented, but here MapStateEntries are modified in place. Should add a comment, though!

Reply:
Please do

Reply:
Added comment!